### ISSUE TITLE
Bug: startpage engine

### ISSUE THREAD

**Issue by iblowmymind** [paulgoio/searxng#30]
**Version of SearXNG, commit number if you are using on master branch and stipulate if you forked SearXNG**
Repository: https://github.com/paulgoio/searxng
Branch: main
Version: 2022.03.05-740e3bc9

**How did you install SearXNG?**

**What happened?**

**How To Reproduce**

**Expected behavior**

**Screenshots & Logs**

**Additional context**

**Technical report**

Error
  * Error: 2 redirects, maximum: 0
  * Percentage: 50
  * Parameters: `('200', 'OK', 'www.startpage.com')`
  * File name: `searx/search/processors/online.py:105`
  * Function: `_send_http_request`
  * Code: `count_error(`

Error
  * Error: 1 redirects, maximum: 0
  * Percentage: 35
  * Parameters: `('200', 'OK', 'www.startpage.com')`
  * File name: `searx/search/processors/online.py:105`
  * Function: `_send_http_request`
  * Code: `count_error(`

Error
  * Error: searx.exceptions.SearxEngineCaptchaException
  * Percentage: 5
  * Parameters: `('CAPTCHA',)`
  * File name: `searx/engines/startpage.py:66`
  * Function: `raise_captcha`
  * Code: `raise SearxEngineCaptchaException(suspended_time=7 * 24 * 3600)`

**Comment by mrpaulblack** [comment]
Duplicate of #29; Closing.

**Comment by mrpaulblack** [comment]
nvm I thought this was a Qwant bug ticket mb; Reopening.

**Comment by mrpaulblack** [comment]
Since I setup some HTTP proxys on my public instance, startpage should work for now. But since startpage is blocked on all other public instances with a CAPTCHA, I dont know for how long tho; Closing.
